get_url/help.md

```diff
@@ -11,8 +11,8 @@ To reduce the number of subsequent requests the Etag and If-Modified-Since field
 
 # Requirements
 
-* Example: Requires an API Key from the product
-* Example: API must be enabled on the Settings page in the product
+* Get the contents of a web page
```

Comment:
Should this be under Key Features instead of Requirements?

Reply:
Yes. Fixed.